test/segment_test.exs

```diff
@@ -5,7 +5,36 @@ defmodule SegmentTest do
 
   test "track debugging" do
     Segment.start_link(@segment_test_key)
-    t = Segment.Analytics.track("343434", "track debugging #{elem(:os.timestamp,2)}")
-    Task.await(t)
+
+    Segment.Analytics.track("user1", "track debugging #{elem(:os.timestamp(), 2)}")
+
+    wait_random()
+
+    Segment.Analytics.identify("user1", %{
+      debug: "identify debugging #{elem(:os.timestamp(), 2)}"
+    })
+
+    wait_random()
+
+    Segment.Analytics.screen("user1", "screen debugging #{elem(:os.timestamp(), 2)}")
+
+    wait_random()
+
+    Segment.Analytics.alias("user1", "user2")
+
+    wait_random()
+
+    Segment.Analytics.group("user1", "group1", %{
+      debug: "group debugging #{elem(:os.timestamp(), 2)}"
+    })
+
+    wait_random()
+
+    Segment.Analytics.page("user1", "page debugging #{elem(:os.timestamp(), 2)}")
+
+    # wait 5 seconds for batcher
+    Process.sleep(5000)
```

Comment:
Instead of doing this, we should make a `flush/0` function or something on the batcher, or something to output all items currently in the queue?

Reply:
yeah that is probably a good idea.